src/gap.c

```diff
@@ -1327,7 +1328,34 @@ Obj FuncRESUME_TIMEOUT( Obj self, Obj state ) {
   return True;
 }
 
- 
+/****************************************************************************
+**
+*F RegisterBreakloopObserver( <func> )
+**
+** Register a function which will be called when the break loop is entered
+** and left. Function should take a single Int argument which will be 1 when
+** break loop is entered, 0 when leaving.
+**
+** Note that it is also possible to leave the break loop (or any GAP code)
+** by longjmping. This should be tracked with RegisterSyLongjmpObserver.
+*/
+
+
+static intfunc signalBreakFuncList[16];
+
+Int RegisterBreakloopObserver(intfunc func)
+{
+    Int i;
+    for(i = 0; i < 16; ++i)
```

Comment:
This code is formatted differently than what we agreed on. Here is what clang-format makes out of it:
```C
Int RegisterBreakloopObserver(intfunc func)
{
    Int i;
    for (i = 0; i < 16; ++i) {
        if (signalBreakFuncList[i] == 0) {
            signalBreakFuncList[i] = func;
            return 1;
        }
    }
    return 0;
}
```